ui/flake-module.nix

```diff
@@ -0,0 +1,83 @@
+{ self, ... }:
+{
+  perSystem = { config, self', inputs', system, ... }:
+    let
+      pkgs = inputs'.nixpkgs.legacyPackages;
+      inherit (config) cat-lib dream2nix;
+      ui =
+        (dream2nix.lib.makeOutputs { source = ./.; settings = [{ subsystemInfo.nodejs = 16; }]; }).packages.danaswap-ui;
```

Comment:
I try to keep danaswap-browser and UI separated, the browser is to provide components (written in Purescript, built with purs-nix) that will be used in UI (written in EJS, built with dream2nix + build.js)

Reply:
Yes, but you don't need to write a build script in javascript in order to bundle the purescript code with the javascript components. So that is why I'm asking

Reply:
Okay I see now what you did, had to look a bit closer. So you build both things independently and then you just copy files around to the right places